rdmd_test.d

```diff
@@ -310,6 +310,15 @@ void runTests()
         assert(res.status == -SIGSEGV, format("%s", res));
     }
 
+    /* https://issues.dlang.org/show_bug.cgi?id=11997- rdmd should search it's
+    binary path for the compiler */
+    string localDMD = buildPath(tempDir(), baseName(compiler));
+    std.file.write(localDMD, "empty shell");
+    scope(exit) std.file.remove(localDMD);
+    res = execute(rdmdApp ~ [modelSwitch, "--force", "--chatty", "--eval=writeln(`Compiler found.`);"]);
+    assert(res.status == 1, res.output);
+    assert(res.output.canFind(`spawn ["` ~ localDMD ~ `",`));
```

Comment:
I think you could do a "positive" test by putting a fake compiler binary in the same directory as rdmd, but it's entirely up to you. Another way would be looking at the output of `--dry-run`, and simply checking that it would have executed the correct compiler.